### ISSUE TITLE
Ignore games in the library view

### ISSUE THREAD

**Issue by Polynomdivision** [lutris/lutris#1456]
Since I enabled Proton in my Steam client, lutris shows me "Proton 3.7 Beta" as an entry in the view that lists all installed games. As I don't want to see my Proton installation, I'd prefer if it was hidden from the list.

As I don't know how required/needed/wanted this feature is, I chose to implement it on my fork and create this issue. If it is wanted enough, then I'd be glad to create a PR.

TL;DR: I don't want to see some games in lutris and will create a PR after some discussion.

**EDIT**: The branch I'm talking about is [this one](https://github.com/Polynomdivision/lutris/tree/feat/ignore-games).

**Comment by strycore** [comment]
yeah, that would be nice to have

**Comment by strycore** [comment]
This has been merged, thanks @Polynomdivision !
